lib/apachetest.pm

```diff
@@ -75,7 +75,8 @@ sub setup_apache2 {
     }
     if ($mode =~ m/SSL|NSS/) {
         assert_script_run "sed -i '/^APACHE_SERVER_FLAGS=/s/^/#/' /etc/sysconfig/apache2";
-        assert_script_run 'echo APACHE_SERVER_FLAGS="SSL" >> /etc/sysconfig/apache2';
+        assert_script_run 'echo APACHE_SERVER_FLAGS=\"SSL\" >> /etc/sysconfig/apache2';
```

Comment:
I think you don't need to escape `"` wihting `''`

Reply:
yes, from the perl side but bash will remove the double quote.

Reply:
ok